esmvaltool/cmorizers/data/formatters/utilities.ncl

```diff
@@ -34,6 +34,26 @@ TUNITS = "days since 1950-01-01 00:00:00"
 ; CMOR FillValue
 FILL = 1.e+20
 
+undef("get_year")
```

Comment:
Could you check this file? It still contains paths from the old framwork (esmvaltool/cmorizers/obs/utilities.ncl). I think this is one reason why the ESACCI-OZONE formatter script doesn't work.

Reply:
Done!